### ISSUE TITLE
FR: Template subfolder variable

### ISSUE THREAD

**Issue by richhayler** [sjelfull/craft3-templateselect#2]
Could you please add the option to output the selected template's full path or subfolder so in a front end template this doesn't have to be hardcoded?

so instead of `{% include '_mysubfolder/' ~ entry.template %}`

maybe `{% set template = entry.template %}`
then `{% include template.filepath %}`
or `{% include template.subfolder ~ template %}`

**Comment by anchovy** [comment]
Big upvote for this.

Another option would to allow the subfolder to be set via a _.env_ variable.


*Application Info*
PHP version | 8.0.8
MySQL 8.0.27
Craft Pro 3.7.31
Yii version 2.0.44
Twig version v2.14.11
Template Select 2.0.2

**Comment by sjelfull** [comment]
This is now available with v5.0.0.

These are the different options now:

### Output the chosen template name:

```twig
{{ entry.fieldHandle }}
```

### Include/Embed the chosen template:

```twig
{% include entry.fieldHandle %}
```

### Include the template including subfolder if set:

```twig
{% include entry.fieldHandle.withSubfolder() %}
```

This is a alias for the following:

```twig
{{ entry.templateWithSubfolder.template(true) }}
```

### Output the subfolder name:

```twig
{{ entry.fieldHandle.subfolder() }}
```

### Output the filename without path:

```twig
{{ entry.fieldHandle.filename() }}
```
